### ISSUE TITLE
Logo in Umbraco Content data source doesn't look correct

### ISSUE THREAD

**Issue by bjarnef** [leekelleher/umbraco-contentment#77]
### What is the current behaviour?

When selecting Umbraco Content as datasource the Umbraco logo is cutted off and looks a bit blurred.
 
<img width="679" alt="chrome_IT5KZd3f6D" src="https://user-images.githubusercontent.com/2919859/110705853-46d1ec80-81f7-11eb-98e4-3532e8cffbae.png">

### What is the expected behaviour?

I guess it should show the full Umbraco logo and max 32x32px or maybe include a SVG version of the logo in the package?

https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/src/assets/img/application/logo_black.png

### Please tell us about your set-up:

- Contentment version: [1.2.0]
- Umbraco version: [8.8.3]
- Umbraco ModelsBuilder is: [enabled]
- Browser: [Chrome Version 88.0.4324.190]

**Comment by leekelleher** [comment]
@bjarnef Thanks raising this. I suspect it's from when the font icons were changed to SVG, (was it v8.8?)

To be honest the use of the Umbraco logo as the icon is a bit of a hack, needs doing properly, e.g. the v8.8 way.

Contentment v1.x is compiling against Umbraco v8.6.x, so that's what I'm currently testing against. v2.x will compile against latest at that point in time, (right now that'll be v8.12).

So for v1.x, if we can get it looking good in both v8.6 and v8.latest, that'd be great. Otherwise I'll hold off until v2.0.

**Comment by leekelleher** [comment]
That's what I meant by it being a hack. By loading in the PNG. But v8.8 must have changed some other CSS rule for icons, that's made it display badly.

I'll take a look. 👍

**Comment by bjarnef** [comment]
Yeah, it could be different things regarding the SVG icons in v8.8 and adjustment to `.umb-node-preview__icon` class.

It seems this part comes from Contentment:

```
[class].icon-umbraco {
    background-image: url(/umbraco/assets/img/application/logo_black.png);
    background-repeat: no-repeat;
    background-size: cover;
}
```

but this work better:

```
[class].icon-umbraco {
    background-image: url(/umbraco/assets/img/application/logo_black.png);
    background-repeat: no-repeat;
    background-size: contain;
}
```

![image](https://user-images.githubusercontent.com/2919859/110710092-8dc2e080-81fd-11eb-83cb-cad9bd5a5d9f.png)

**Comment by leekelleher** [comment]
@bjarnef Thank you for following up with the CSS amend. 👍 

I've tried out the `background-size: contain;` change on v8.6.6 and v8.11.1, both display fine.  Although the vertical alignment isn't centre on v8.11.1, but I'm okay with compromising on that for Contentment v1.3+.

With upcoming Contentment v2.0, I'll review the icon usage. I'm sure there are more built-in Umbraco goodies to leverage since v8.8.

**Comment by bjarnef** [comment]
@leekelleher yes the alignment is a bit off, but it seems `background-position: left center;` works or in case it had a longer description (multiline) `background-position: left 10px;`

**Comment by leekelleher** [comment]
I've started work on v2.0, and figured out how to add a custom SVG icon. Fixed in 2a7bd7f99cbf9365ee8a5dac50cbd557440c8394. 🙌
